act-framework.bs

```diff
@@ -42,10 +42,11 @@ Rule Description {#structure-description}
 
 Each ACT Rule MUST have a description that:
 
-* Is in plain language;
-* Explains the accessibility requirement being tested;
-* Lists any limitations and assumptions for the test;
-* Explains the failure conditions and their reasons.
+* Is in plain language and provides a brief description of what the rule does.
+* Explains the accessibility requirement being tested such as the WCAG SC or the technique the rule maps to; For example WCAG 2.0 Technique H67.
+* Lists any limitations and assumptions for the test, the test environment, technologies being used or the subject being tested; For example: A rule for 1.4.1: Use of Color has to make an assumption with CSS-properties are used to make a link visually evident. Typically something like background, border, color, font, or text-decoration.
```

Comment:
Editorial: '...has to make an assumption **that** CSS properties', don't hyphenate CSS properties, change 'with' to 'that'.  The last sentence is missing a verb, so suggest saying 'Typical examples are: background, border, color, font, or text decoration'.  Text decoration should not be hyphenated.